script.js

```diff
@@ -1,5 +1,7 @@
 var previousTab = 'home-page';
-document.addEventListener("click", deleteLogo);
+
+let slideIndex = 1;
```

Comment:
I meant to mention this on previous PRs, but it's rarely a good idea to have global variables like this.  slideIndex will only be used by a few functions and only relates to the slides. 

Use a closure to contain the variable and move your functions inside that to limit the scope.  Something like

```
function Slideshow() {
  let slideIndex = 1;

  const showSlides = (n) => {
    // function body here
  };
 
  const currentSlide = (n) => {
    // function body here
  }

 const plusSlides = (n) => {
   // function body here
  }

  return { plusSlides, currentSlide };
};
```
And then in the global scope
```
const slideshow = Slideshow();

slideshow.currentSlide(2); // or however you'd want to use it
```

It's not uncommon to contain ALL Javascript within a "top level" closure to avoid having ANYTHING in the global scope at all, but for now we can just focus on at least grouping obvious bits of code together